File: src/main/golang/leetcode/editor/cn/[70]Climbing Stairs.go
package lc

//leetcode submit region begin(Prohibit modification and deletion)
func climbStairs(n int) int {
	n0, n1 := 1, 1
	for i := 2; i <= n; i++ {
		n0, n1 = n1, n0+n1
	}
	return n1
}

//leetcode submit region end(Prohibit modification and deletion)

File: src/main/golang/leetcode/editor/cn/[70]Climbing Stairs.py
# leetcode submit region begin(Prohibit modification and deletion)
class Solution:
    def climbStairs(self, n: int) -> int:
        n0, n1 = 0, 1
        for i in range(1, n + 1):
            n0, n1 = n1, n0 + n1
        return n1

# leetcode submit region end(Prohibit modification and deletion)
